src/ol_infrastructure/applications/ocw_site/Pulumi.yaml

```diff
@@ -0,0 +1,5 @@
+name: ol-infrastructure-ocw_site-application
+runtime: python
+description: OCW next-generation site contents stored in S3 buckets and hosted on the CDN
```

Comment:
We should probably avoid using the `next-generation` terminology as much as possible.

Reply:
Fixed in a3fae49